platform/lf_nrf52_support.c

```diff
@@ -104,6 +111,18 @@ void calculate_epoch_offset() {
  */
 uint32_t _lf_previous_timer_time = 0u;
 
+void lf_timer_event_handler(nrf_timer_event_t event_type, void *p_context)
+{
+    // interrupted by timer first
+    if (event_type == NRF_TIMER_EVENT_COMPARE2)
+        _lf_interrupted = false;
+    // overflow handle
+    if (event_type == NRF_TIMER_EVENT_COMPARE3) {
+        nrfx_timer_clear(&g_lf_timer_inst);
```

Comment:
Is clearing really the right thing to do? I'm guessing that NRF_TIMER_EVENT_COMPARE3 means that the timer hit its maximum and stopped? If it actually overflowed, then clearing it will introduce a timing offset, which seems like an error to me.

Reply:
makes sense, removed the clear